reference/release-notes/1.21.2.md

```diff
@@ -0,0 +1,29 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* WiFi support for AAOS images.<!--AC-2211-->
+* AAOS-related enhancements:
+   - Fixing the `systemui` crash <!--AC-2267-->
+   - Fixing Bluetooth crash in the AAOS launcher <!--AC-2265-->
+   - Session crash when opening installed app ([LP 2054443](https://bugs.launchpad.net/anbox-cloud/+bug/2054443)) <!--AC-2297-->
```

Comment:
Looking at this, should it be in the bug fixes section instead?

Reply:
Ideally, I would have listed it in the bug fixes section but was just trying to group together the AAOS related stuff.
Also, the first two didn't have bugs that can be linked.

Reply:
I have now moved them to bug fixes section, it is fine if they don't have LP bugs linked as I checked and we have listed some internal bugs without a link in the past.